GiEnJul/ClientApp/src/components/InputFields/Validators/InputValidator.tsx

```diff
@@ -38,10 +38,10 @@ interface InputValidatorProps {
     autoFocus?: boolean,
     size?: any,
     multiline?: boolean,
-    rowsMax?: number | string
+    rowsMax?: number | string,
+    placeholder?: string,
 
     options?: {value: any, text: string}[],  // Only for "type=select".
-    isMobile?: boolean,   // Only for "type=select". If true: select dropdown uses native dropdown, which is better for mobile.
```

Comment:
I think textFields also should have a layout that is optimized for mobile. For example that the placeholderText is removed if the placeholderText exceeds the input field. 
This could be another issue :)
![image](https://user-images.githubusercontent.com/37541100/124563259-2c73ec80-de40-11eb-8b2a-e6ad3a056a8b.png)

Reply:
Hmm good point. The text should probably also be shortened in this case. Could it be an idea to wait with this until the design for it is finished? Khoi might have a nice way to deal with this problem.

Reply:
Sounds good, I will create an issue about this.

Reply:
Is it supposed to be optimized for phone at all. Is this site not just for admins that register families?

Reply:
That's true. This component (InputValidator) is also used in the giver form though. A nice solution could be to truncate the placeholders, and/or set overflow to none.